src/components/JaegerIntegration/TracesComponent.tsx

```diff
@@ -379,8 +262,9 @@ class TracesComponent extends React.Component<TracesProps, TracesState> {
                     </Toolbar>
                   </RenderHeader>
                   <JaegerScatter
-                    fixedTime={!this.state.adjustTime}
-                    traces={this.filterTraces()}
+                    fixedTime={false}
```

Comment:
I guess only fixedTime was used for Jaeger I think.
Perhaps then that parameter could be removed, but it's fine.

Reply:
+1, I've removed it